templates/msp430/exp430g2/mods.config

```diff
@@ -17,7 +17,7 @@ configuration conf {
 	include embox.kernel.cpu.bkl_api
 
 	include embox.compat.libc.stdio.print(support_floating=0)
-	include third_party.lib.NoOpenLibm
+	include third_party.lib.libm_none
```

Comment:
I think it should be `embox.compat.libc.math_libm_none` (same with other templates).